pytest_inmanta/module/init.py

```diff
@@ -113,3 +113,19 @@ def get_desired_value(exporter, resource):
 @handler.provider("unittest::IgnoreResource", name="ignore")
 class IgnoreResourceHandler(handler.CRUDHandler):
     pass
+
+
+@resources.resource(
+    "unittest::IgnoreResourceInIdAttr", id_attribute="name", agent="agent"
+)
+class IgnoreResourceInIdAttr(resources.PurgeableResource):
+    fields = ("name", "desired_value")
+
+    @staticmethod
+    def get_name(exporter, resource):
+        raise resources.IgnoreResourceException()
+
+
+@handler.provider("unittest::IgnoreResourceInIdAttr", name="ignore")
+class IgnoreResourceInIdAttrHandler(handler.CRUDHandler):
+    pass
```

Comment:
Something you may be unaware of: You can put multiple `@handler.provider` annotations above a single class. So you could just as well have reused the `IgnoreResourceHandler` class for this.

Reply:
Indeed, I didn't know. I'll change it